src/elements/hx-tabset/index.spec.js

```diff
@@ -329,5 +329,32 @@ describe('<hx-tabset> component tests', () => {
             expect(firstTabCount).to.equal(0);
             expect(secoundTabCount).to.equal(1);
         });
+
+        it.skip(`[FEATURE] should Not able to Navigate between one or more tab panels in angular ${template}`, async () => {
+            const mockupName = `
+                <hx-tabset>
+                    <hx-tablist>
+                        <hx-tab>First</hx-tab>
+                        <hx-tab>Second</hx-tab>
+                        <hx-tab>Third</hx-tab>
+                    </hx-tablist>
+                    <hx-tabcontent>
+                        <hx-tabpanel>...</hx-tabpanel>
+                        <hx-tabpanel>...</hx-tabpanel>
+                        <hx-tabpanel>...</hx-tabpanel>
+                    </hx-tabcontent>
+                </hx-tabset>`;
+
+            const fragment = /** @type {HXTabsetElement} */ await fixture(mockupName);
+
+            // Selected tab
+            const selectedTab = fragment.querySelector('hx-tab:nth-child(2)');
+
+            // Data tabpanel
+            const selectedContent = fragment.querySelector('hx-tabpanel:nth-child(2)');
+
+            expect(selectedTab).to.not.null;
```

Comment:
Please change to:

```js
expect(selectedTab).to.not.be.null;
```

Reply:
Also, please attach console.log of selectedTab.

Reply:
Changes have been done, Please review